content/en/continuous_integration/pipelines/azure.md

```diff
@@ -32,17 +32,61 @@ further_reading:
 
 The Datadog integration for [Azure Pipelines][1] works by using [service hooks][2] to send data to Datadog.
 
+1. Install the [Datadog CI Visibility][8] extension from the Azure Marketplace.
+
 1. For each project, go to **Project settings > Service hooks** in Azure DevOps and select `Create subscription` (green plus icon).
```

Comment:
This should be 2. and so on right? I see the UI displays it well, but let's clean it here

Reply:
The number was off but the rest are not 1